src/nvme/ioctl.h

```diff
@@ -1133,6 +1133,15 @@ int nvme_get_log_zns_changed_zones(int fd, __u32 nsid, bool rae,
 int nvme_get_log_persistent_event(int fd, enum nvme_pevent_log_action action,
 				  __u32 size, void *pevent_log);
 
+/**
+ * nvme_media_unit_stat_log() -
+ * &fd: File descriptor of nvme device
+ * &domid: Domain Identifier
+ * &mus: User address to store the media unit status log
+ */
+
```

Comment:
- no new line between the doc and the function.
- please align the documentation like the other function have.

Reply:
Addressed
Thanks @igaw